src/CappedOracle.sol

```diff
@@ -11,7 +12,9 @@ contract CappedOracle {
     int256       public immutable maxPrice;
 
     constructor(address _source, int256 _maxPrice) {
-        require(_maxPrice > 0, "CappedOracle/invalid-max-price");
+        // We require 8 decimals as AaveOracle assumes this
```

Comment:
`// 8 decimals required as AaveOracle assumes this`

Reply:
changed